src/ui/post_card/big_post_card/big_post_card.tsx

```diff
@@ -0,0 +1,137 @@
+import { useState } from 'react';
+import { styled } from 'styled-components';
+import { PostCardModel } from '../post-card.model';
+
+
+type BigPostCardProps = {
+  postCard: PostCardModel;
+};
+
+export const BigPostCard: React.FC<BigPostCardProps> = (
+  props: BigPostCardProps
+) => {
+  const [isLiked, setIsLiked] = useState<boolean>(false);
+  const [isDisliked, setIsDisliked] = useState<boolean>(false);
+  const [amountOfLikes, setAmountOfLikes] = useState<number>(props.postCard.likes_amount);
+  const [isSaved, setIsSaved] = useState<boolean>(false);
+
+  function dislike() {
+    if(isLiked){
+      setAmountOfLikes(amountOfLikes - 1)
+    }
+    setIsDisliked(!isDisliked);
+    setIsLiked(false);
+  }
+  function like() {
+    setIsLiked(!isLiked);
+    setIsDisliked(false);
+    setAmountOfLikes(isLiked ? amountOfLikes - 1 : amountOfLikes + 1);
+  }
+  return (
+    <BigPostCardWrapper>
+      <MainWrapper>
+        <CardtextWrapper>
+          <PostCardDate>{props.postCard.date}</PostCardDate>
+          <PostCardTitle>{props.postCard.title}</PostCardTitle>
+          <PostCardText>{props.postCard.text}</PostCardText>
+        </CardtextWrapper>
+        <CardImageWrapper>
+          <img src={props.postCard.image} alt="#"></img>
+        </CardImageWrapper>
+      </MainWrapper>
+      <IconWrapper>
+        <LikeWrapper>
+          <LikeIcon onClick={() => like()}> {isLiked ? <i className="fa-solid fa-thumbs-up"></i> :<i className="fa-regular fa-thumbs-up"></i>}</LikeIcon>
+          <AmountOfLikes>{amountOfLikes}</AmountOfLikes>
+          <DislikeIcon onClick={() => dislike()}> {isDisliked ? <i className="fa-solid fa-thumbs-down"></i> : <i className="fa-regular fa-thumbs-down"></i>}</DislikeIcon>
+        </LikeWrapper>
+        <SaveIcoonWrapper>
+          <BookmarkIcon onClick={() =>setIsSaved(!isSaved)}> {isSaved ? <i className="fa-solid fa-bookmark"></i> : <i className="fa-regular fa-bookmark"></i>}</BookmarkIcon>
+          <i className="fa-solid fa-ellipsis"></i>
+        </SaveIcoonWrapper>
+      </IconWrapper>
+    </BigPostCardWrapper>
+  );
+};
+
+export const BigPostCardWrapper = styled.div`
+  width: 610px;
+  height: 325px;
+  background-color: #f3f3f3;
+  padding: 15px 20px;
+  border-bottom: 1px solid #cfd0d1;
+`;
+export const MainWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+export const CardtextWrapper = styled.div`
+width: 65%;
+
+`;
+export const PostCardDate = styled.div`
+color: #8B8A90;
+font-size: 14px;
+line-height: 15px;
+`;
+
+export const PostCardTitle = styled.h2`
+color: #403F44;
+line-height: 30px;
+font-size: 28px;
+margin:10px 0;
+`;
+
+export const PostCardText = styled.div`

```

Comment:
do not export components are not supposed to be used outside, it can lead to autocomplete problems